maven-core/src/main/java/org/apache/maven/project/DefaultProjectBuilder.java

```diff
@@ -121,6 +121,7 @@
 public class DefaultProjectBuilder implements ProjectBuilder {
     public static final String BUILDER_PARALLELISM = "maven.projectBuilder.parallelism";
     public static final int DEFAULT_BUILDER_PARALLELISM = Runtime.getRuntime().availableProcessors() / 2 + 1;
+    private static final String POM_4_0_0 = "4.0.0";
```

Comment:
Why not `MODEL_VERSION_4.0.0` as above?

Reply:
That one isn't used anymore, I'll remove it.